packages.yml

```
@@ -1,3 +1,3 @@
 packages:
   - package: dbt-labs/dbt_utils
-    version: [">=0.8.0", "<1.0.0"]
+    version: [">=1.0.0", "<2.0.0"]
```

In order to support dbt 1.0.0 and above, we need to support older versions of dbt-utils as well:
```suggestion
    version: [">=0.8.0", "<2.0.0"]
```